README.md

```diff
@@ -1 +1,98 @@
+
+# Project Name : COVID GLOBAL METRICS
+
+>covid-metrics is a website app for user's to get real time covid cases through the world.
+![Screenshot from 2022-06-30 04-57-25](https://user-images.githubusercontent.com/56429354/176659513-c51e5e68-2755-48f7-83f3-f42e787c5720.png)
+
+[LIVE DEMO](https://covid-metric.netlify.app/)
+
+## Built With
+
+- React Js 
+- Redux 
+- React Router 
+- GitFlow
+
+
+## Development set up
+
+Clone Repository using
+
+`git clone git@github.com:gedewon/covid-metrics.git`
+
+OR using HTTPS
+
+`git clone https://github.com/gedewon/covid-metrics.git`
+
+move into project directory
+
+`cd covid-metrics`
+
+Install Project dependencies using `NPM`
+
+`npm install`
+
+launch project locally with
+
+`npm start`
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+## Author
+
+👤 **Gedewon**
+
+- GitHub: [@githubhandle](https://github.com/gedewon)
+- Twitter: [@twitterhandle](https://twitter.com/gedewon)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/gedewon)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Gedewon/covid-metrics/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Kindly add an Acknowledgements section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/)